acquia_lift.admin.wizard.inc

```diff
@@ -605,6 +676,28 @@ function acquia_lift_personalize_campaign_wizard_review_alter(&$form, &$form_sta
     $form['review']['warnings_column']['warnings']['#type'] = 'container';
     $form['review']['warnings_column']['warnings']['#theme'] = 'acquia_lift_card';
   }
+  $form['#submit'] = array(
+    'acquia_lift_personalize_campaign_wizard_review_submit'
+  );
+}
+
+/**
+ * Submit handler for the review form.
+ */
+function acquia_lift_personalize_campaign_wizard_review_submit($form, &$form_state) {
+  $agent_data = $form['#agent'];
+  module_load_include('inc', 'acquia_lift', 'acquia_lift.admin');
+  module_load_include('inc', 'acquia_lift', 'acquia_lift.batch');
+
+  try {
+    $agent_data->tests_to_delete = acquia_lift_implement_test_structure($agent_data);
+    acquia_lift_batch_sync_tests(array($agent_data->machine_name => $agent_data));
+    // If the "Start" button was pressed then start the campaign.
```

Comment:
It's possible to save this form on the review page without hitting the start button.  Should be checked like we do in the base personalize module: https://github.com/acquia/personalize_module/blob/workflow/personalize.admin.campaign.inc#L1391